### PR TITLE
Add yacht

### PR DESCRIPTION
**Pull request by BNAndras**
For #48in24

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -721,6 +721,14 @@
           "strings"
         ]
       },
+      {
+        "slug": "yacht",
+        "name": "Yacht",
+        "uuid": "32341692-02df-48a0-b696-1cfe93dc6862",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```
Comment (ErikSchierboom) [inline config.json:730]: Maybe make this a `3` difficulty?

Comment (ErikSchierboom) [reply config.json:730]: Right. What does Julia do?

Comment (BNAndras) [reply config.json:730]: Apparently not that so I can bump the difficulty.

Comment (BNAndras) [reply config.json]: Done.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**